commit 1be0681cc4bcc006369e69d90dc4439eaa6f58d5
Author: Stefan Schantl <stefan.schantl@ipfire.org>
Date:   Wed Jul 29 18:50:20 2020 +0200

    location: Provide a return code if the database does not need to be
    updated.
    
    Signed-off-by: Stefan Schantl <stefan.schantl@ipfire.org>

diff --git a/src/python/location.in b/src/python/location.in
index 5211b28..8cdd140 100644
--- a/src/python/location.in
+++ b/src/python/location.in
@@ -416,7 +416,7 @@ class CLI(object):
 					_("The datase has recently be updated recently (%s)") % \
 						format_timedelta(now - t),
 				)
-				return
+				return 3
 
 		# Fetch the timestamp we need from DNS
 		t = location.discover_latest_version()
